### DISCUSSION TITLE
Resume Score

### DISCUSSION THREAD

**Discussion by lassmara** [recodehive/resume-pitch#104] [Q&A]
I am applying for a Software Developer role, and I’ve attached my resume along with my LinkedIn profile. Could you kindly provide feedback on my resume, specifically focusing on the relevance and impact of the projects listed? Additionally, I’d like to understand if there is any bias in the resume, and how well it might perform with Applicant Tracking Systems (ATS).

linkedin: https://www.linkedin.com/in/sai-sruthi-maraboina/?trk=public-profile-join-page

[B21202_SaiSruthi Maraboina.pdf](https://github.com/user-attachments/files/17085005/B21202_SaiSruthi.Maraboina.pdf)

**Comment by sanjay-kv** [comment]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
